Update RELEASE_PROCESS (#6754)
wallabag/wallabag: RELEASE_PROCESS.md

#### Create a new release on GitHub

- [Create the new release on GitHub](https://github.com/wallabag/wallabag/releases/new) by targetting the `master` branch or any appropriate branch (for instance backports).

> - Update [website](https://github.com/wallabag/website) to change downloads, MD5 sum, releases, the new blog post and the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.

Reviewer: ```suggestion
- Update [website](https://github.com/wallabag/website) to change MD5 sum and create the release blog post (based on the changelog).
```

#### Create a new release on GitHub

- [Create the new release on GitHub](https://github.com/wallabag/wallabag/releases/new) by targetting the `master` branch or any appropriate branch (for instance backports).
- Update [website](https://github.com/wallabag/website) to change downloads, MD5 sum, releases, the new blog post and the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.
- Update Dockerfile https://github.com/wallabag/docker (and create a new tag)

> - [Create the new Docker release on GitHub](https://github.com/wallabag/docker/releases/new). 

Reviewer: This is already explained line 21

Author: Oops, sorry.
